### PR TITLE
[W6.6h][T16-B2]Lin Jiachao

### PR DESCRIPTION
**Pull request by lincredibleJC**
Add Rename tag use case

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -87,14 +87,46 @@ Use case ends.
...
+
+> 3b1. AddressBook will remind the user to add the new name <br>
+  Use case resumes at step 2
+   
+5.a The user writes confirms no
+
+> 5a1. Use case ends
+
+5.b The user types enters something other than yes and no
+
```
Comment (brandonyeoxg) [inline doc/DeveloperGuide.md:120]: Good extension!

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -87,14 +87,46 @@ Use case ends.
...
+> 5a2. AddressBook will notify user of invalid input <br>
```
Comment (brandonyeoxg) [inline doc/DeveloperGuide.md:121]: The numbering has problem? It should be `5b1` right?

**Review by brandonyeoxg — COMMENTED** [review]
Accepting this PR without a review because another of your PRs for this tutorial was reviewed in detail.

File: doc/DeveloperGuide.md (hunk 2 above)
Comment (lincredibleJC) [reply doc/DeveloperGuide.md:121]: Yup, I must have forgotten to change it. I've just added a commit to fix that. Do you want to reopen this PR to see it?

**Pull request closed without merging**